packages/migrate/src/__tests__/DbExecute.test.ts

```diff
@@ -400,6 +400,11 @@ COMMIT;`,
       ctx.fixture('schema-only-postgresql')
       expect.assertions(2)
 
+      // In CI, sometimes 5s is not enough
+      if (process.env.CI) {
+        jest.setTimeout(10_000)
```

Comment:
Is this test really more heavy than the other ones in this file or is it just the only one that timed out so far? Would changing the timeout for the whole test suite be more appropriate maybe?

Reply:
Ah, I see, maybe DNS resolution takes too long before it gives up or something like that. Sounds good then.